course-structure.json

```
@@ -108,95 +90,93 @@
                 {
-                    "title": "Native Solana Program Development",
-                    "slug": "basic-solana-program-development",
+                    "title": "Introduction to Solana tokens",
+                    "slug": "introduction-to-solana-tokens",
                     "lessons": [
                         {
-                            "title": "Serialize program data",
-                            "slug": "serialize-instruction-data",
-                            "lab": "Build a React frontend to submit movie reviews"
-                        },
-                        {
-                            "title": "Deserialize program data",
-                            "slug": "deserialize-custom-data",
-                            "lab": "Build a React frontend to show movie reviews"
-                        },
-                        {
-                            "title": "Page, Order, and Filter program data",
-                            "slug": "paging-ordering-filtering-data",
-                            "lab": "Adding paging, ordering and searching to the movie reviews frontend"
-                        },
-                        {
-                            "title": "Hello World in native",
-                            "slug": "hello-world-program",
-                            "lab": "Create and deploy a native 'hello world' onchain program"
-                        },
-                        {
-                            "title": "Handle Instruction Data",
-                            "slug": "deserialize-instruction-data",
-                            "lab": "Make an onchain program that recieves movie reviews and deserialises them"
-                        },
-                        {
-                            "title": "State Management",
-                            "slug": "program-state-management",
-                            "lab": "Make an onchain program that stores movie reviews"
+                            "title": "Create tokens with the Token Program",
+                            "slug": "token-program",
+                            "lab": "Mint, transfer and burn tokens"
                         },
                         {
-                            "title": "Basic Security and Validation",
-                            "slug": "program-security",
-                            "lab": "Add security checks and updates to the movie reviews program"
+                            "title": "Create Solana NFTs With Metaplex",
+                            "slug": "nfts-with-metaplex",
+                            "lab": "Mint NFTs, update their metadata, and associate with a collection"
                         }
                     ]
                 },
                 {
-                    "title": "Intermediate Solana Program Development",
-                    "slug": "intermediate-solana-program-development",
+                    "title": "Token Extensions Program",
```

We want a task-based title here. Low effort would be 'Using the Token Extensions Program".
```suggestion
                    "title": "Using the Token Extensions Program",
```